modules/translation/translation.go

```diff
@@ -38,6 +36,7 @@ func AllLangs() []LangType {
 
 // InitLocales loads the locales
 func InitLocales() {
+	i18n.Flush()
```

Comment:
The change seems unrelated with the topic?

Reply:
Dont think it's worth its own pull

Reply:
will make a own pull ...

Reply:
->  #15073